src/main/webapp/WEB-INF/spring/task-scheduler-config.xml

```diff
@@ -0,0 +1,11 @@
+<beans xmlns="http://www.springframework.org/schema/beans"
```

Comment:
This and the changes in web.xml are about enabling springs Scheduling framework.